ingestion/transformers/assign_cadet_domains.py

```diff
@@ -74,24 +75,25 @@ def _get_domain_mapping(self, manifest) -> PatternDatasetDomainSemanticsConfig:
             node_info = nodes[node]
             if node_info["resource_type"] != "model":
                 continue
-            domain, escaped_urn_for_regex = self._convert_cadet_manifest_table_to_datahub(node_info)
+            domain, escaped_urn_for_regex = convert_cadet_manifest_table_to_datahub(node_info)
             domain_mappings[escaped_urn_for_regex] = [domain]
 
         pattern_input = {"domain_pattern": {"rules": domain_mappings}}
 
         return PatternDatasetDomainSemanticsConfig.parse_obj(pattern_input)
 
-    def _convert_cadet_manifest_table_to_datahub(self, node_info: dict) -> Tuple[str, str]:
-        domain = node_info.get("fqn", [])[1]
-        node_table_name = node_info.get("fqn", [])[-1]
-
-        # In CaDeT the convention is to name a table database__table
-        node_table_name_no_double_underscore = node_table_name.replace("__", ".")
-        urn = builder.make_dataset_urn_with_platform_instance(
-                platform="dbt",
-                platform_instance="cadet.awsdatacatalog",
-                name=node_table_name_no_double_underscore,
-            )
-        escaped_urn_for_regex = re.escape(urn)
+def convert_cadet_manifest_table_to_datahub(node_info: dict) -> Tuple[str, str]:
+    domain = node_info.get("fqn", [])[1]
```

Comment:
here as well I think we should have some error handling/validation of the FQN, just in case CaDeT ever diverge from our assumptions about naming

Reply:
I added a check for the table name, not sure how we'd check other parts of the FQN